README.md

```diff
@@ -23,7 +23,14 @@ To build and compile the project, follow these steps:
 - Run the following command to build the JAR file: `mvn clean package`
 
 ## Usage
-- To start the program with the following command: `java -jar <your-project>.jar`
+You need to copy input file to the same directory which you move your project.jar.
```

Comment:
It important to use passive voice for writing the technical docs.
Do not use `You`.

Rewrite it like:
`The input file should be located in the same directory with the JAR file.`

Reply:
fix